### PR TITLE
Powershell snippet

### PR DESCRIPTION
**Pull request by kiapanahi**
Addresses #1452

### CODE REVIEW

File: .github/wiki/Getting-Started.md [hunk 1]
```diff
@@ -15,38 +16,50 @@ Onefetch is installed, then what?
...
 ```
-By [**@quazar-omega**](https://github.com/quazar-omega)
+
+### Automatic repo detection and running
+
+If you want to automate the detection and running of `onefetch` every time you `cd` into a repository you can leverage one of the methods below:
+
+#### 1. Bash / Zsh
+
+> By [**@quazar-omega**](https://github.com/quazar-omega)
```
Comment (spenserblack) [inline .github/wiki/Getting-Started.md:31]: If it's not too much trouble, can you undo these quotes (`> `)? I guess they were added for emphasis. My concern is that they get rendered to `<blockquote>` in HTML, which can convey unintended semantic meaning and be harmful for screen readers.

**Review by spenserblack — COMMENTED** [review]
Looks great! I appreciate that you went the extra mile and added some more description to the scripts.

File: .github/wiki/Getting-Started.md (hunk 1 above)
Comment (kiapanahi) [reply .github/wiki/Getting-Started.md:31]: sure. good point

**Review by spenserblack — APPROVED** [review]
Looks like this was formatted with a tool like `prettier`? Don't worry about it for this PR, but for future reference it can be helpful to make formatting commits separate from any other changes. This allows formatting commits to be registered in a `.git-blame-ignore-revs` file ([docs](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view)). `.git-blame-ignore-revs` is especially helpful for ignoring automated formatting so you can focus on "real" changes in a blame view. But you don't want to add a commit to `.git-blame-ignore-revs` if it also contains good, non-formatting changes :slightly_smiling_face:

**Comment by spenserblack** [comment]
@o2sh Because the CI runs are required before merge, and they only get run on code changes, I can't merge documentation PRs :sweat_smile: Would you want to merge this?

**Pull request merged**